Autopilot > LoS: UI issues

vkrasnovyd opened the pull request:
Resolve #3846 

1. Fixed: the space to the left of the individual elements is not used in multiple sections:
1.1 In Autopilot: list of speakers, list of previuos speakers.
1.2 In polls > Poll detail page > Candidates list.

2. The horizontal scrollbar appeared because the components of Autopilot page were put inside the container 'content-container' that added margins on the sides. Removed this container to let the content fully adjust to the screen size.
The buttons 'Add me' and 'Point of order' switched in one line indeed.
Note: used poll detail page as reference where 'content-container' wasn't used.

Elblinator reviewed (changes requested):
The placment of the structure level card changed, please change it back:
If the screen is 1050+px wide then the structure level card should be right from the other contents on the autopilot
(note that I configureted my autopilot to only show the LoS and the speaking times card)
**wanted appearance:**
![Screenshot_20240802_105711](https://github.com/user-attachments/assets/bc5f5216-eb17-45e3-af92-ed280880a476)

**currently:**
![Screenshot_20240802_105721](https://github.com/user-attachments/assets/c187d615-66d6-4358-b404-44710a325eb3)
If the other things would be visible in my autopilot then the speaking times card would be below all of them


**How to activate to structure level card:**
1. navigate to settings > List of speakers > third to last point 
2. activate the "Default speaking time contingent for parliamentary groups(structure levels) in seconds"
3. project an object with a LoS onto the Autopilot (e.g. a motion)
4. navigate to participants > three-dot-menu > structure levels
5. create a structure level
6. assign a participant a structure level
7. navigate to the autopilot
8. put someone with a structure level onto the LoS


The rest of the changed appearance is fine :)

vkrasnovyd commented:
It turned out that the removed container was needed to keep timer in the separate column right on big screen. Reverted this change.

The 'display: flex' property inside this container however didn't let the blocks of speakers info properly adjust to small screens. Changed it to 'display: block' for phones only.

bastianjoel commented on client/src/styles.scss:
```diff
@@ -289,10 +289,9 @@ button.mat-mdc-menu-item.selected {
-    .mat-expansion-panel-body {
-        padding: 0 !important;
-    }
+.mat-expansion-panel-body {
+    padding: 0 !important;
+    margin-bottom: 16px;
```
I am not sure if this change was intended. If not please revert.  

```suggestion
.queue {
    .mat-expansion-panel-body {
        padding: 0 !important;
    }
```

bastianjoel replied:
I also meant the removal of the wrapping `.queue`.

vkrasnovyd commented:
Added one more change for better space usage: removed side margins for 'add-self-button' (for small screens only). Now buttons are aligned with the list of speakers on the left side.
